src/pages/api/airtable.ts

```diff
@@ -0,0 +1,39 @@
+import airtable from 'airtable';
+import type { NextApiRequest, NextApiResponse } from 'next';
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method === 'POST') {
+    try {
+      const { helpful, feedback, email } = req.body;
+      const url = req.headers.referer;
+      airtable.configure({
```

Comment:
Because this is a front-end project it looks a bit bad to use airtable here. We could use vercel analytics and just trigger custom events?